apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java

```diff
@@ -149,6 +154,11 @@ private void initializeAutoUpdatePropertiesFeature(ConfigurableListableBeanFacto
     }
   }
 
+  private Boolean isOverrideSystemProperties(ConfigurableEnvironment environment) {
+    return environment.getProperty(ApolloClientSystemConsts.APOLLO_OVERRIDE_SYSTEM_PROPERTIES, Boolean.class, true)
```

Comment:
The same as above.

Reply:
if we don't use `configUtil.isOverrideSystemProperties()` in PropertySourcesProcessor.java, How to check this properties from `/META-INF/app.properties` ?
add a APOLLO_SYSTEM_PROPERTIES array in PropertySourcesProcessor.java, like ApolloApplicationContextInitializer did?

Reply:
oh, it looks I had some typo in previous comments, what I mean is to use `configUtil.isOverrideSystemProperties()`